x-pack/plugins/ml/public/components/anomalies_table/entity_cell.js

```diff
@@ -17,7 +17,7 @@ import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
 /*
  * Component for rendering an entity cell in the anomalies table, displaying the value
  * of the 'partition', 'by' or 'over' field, and optionally links for adding or removing
- * a filter on this entity.
+ * a filter on this entity.  It can be used for any entity which can be added to a filter.
```

Comment:
Or possibly `Component for rendering an entity in the anomalies table, displaying the value
 of the entity, such as a partitioning or influencer field value, and optionally links for 
adding or removing a filter on this entity.`